src/core/util/systemcapabilities.cpp

```diff
@@ -71,121 +62,25 @@ void SystemCapabilities::retrieveDynamicInfo() {
 const util::BuildInfo& SystemCapabilities::getBuildInfo() const { return buildInfo_; }
 
 bool SystemCapabilities::lookupOSInfo() {
-#ifdef IVW_USE_SIGAR
```

Comment:
We should remove these functions unless we replace the functionality. Will be misleading to be able to request the info without it ever being available.

Reply:
mmm removed SystemCapabilities